common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/core/validation/InetAddressValidator.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Yes, it is code copied from Apache Validator and Fred told me this was allowed on a case by case because I can't find this artifact in the target platform